src/index.js

```diff
@@ -0,0 +1,13 @@
+const scores = [
+  { name: 'Jim', score: '30' },
+  { name: 'Ntare', score: '40' },
+];
+
+const scoresContainer = document.getElementById('scores-list');
+
+scores.forEach((score) => {
+  const indivScore = `${score.name}: ${score.score}`;
+  const scoreItem = document.createElement('li');
+  scoreItem.innerHTML = indivScore;
+  scoresContainer.append(scoreItem);
+});
```

Comment:
- [ ] good job changing implementing JS 👍  however part of the assignment is to Use ES6 modules to write modular JavaScript with proper ES6 syntax. which means that you should make use of arrow functions instead of the function keyword.
- [ ] also when building the project and running it I get a blank page, it seems the javascript file is not being built.

Reply:
Hi Jim,
You're right, you didn't use the function keyword anywhere, however you are not using arrow functions either, which is part of the goal of the assignment

Reply:
I used the arrow function in the forEach helper.

Reply:
But I've fixed the build issue, It's now working fine.

Reply:
I replied.